clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/AutoScalingWorker.groovy

```diff
@@ -217,6 +221,17 @@ class AutoScalingWorker {
     if (suspendedProcesses) {
       autoScaling.suspendProcesses(new SuspendProcessesRequest(autoScalingGroupName: asgName, scalingProcesses: suspendedProcesses))
     }
+    if (disabledGroupMetrics) {
```

Comment:
No, you're right. This was an oversight on my part. We don't need to support disabling metrics on ASG creation.